### ISSUE TITLE
[BUG] BinaryOperatorSpacesFixer does not add whitespace around equal sign

### ISSUE THREAD

**Issue by buffcode** [FriendsOfPHP/PHP-CS-Fixer#2411]
Configuration:
```php
return PhpCsFixer\Config::create()
    ->setRiskyAllowed(true)
    ->setCacheFile(__DIR__ . '/.php_cs.cache')
    ->setRules(array(
        '@Symfony'                => true,
        '@PHP70Migration'         => true,
        'ordered_imports'         => true,
        'phpdoc_order'            => true,
        'phpdoc_summary'          => false,
        'phpdoc_separation'       => false,
        'phpdoc_var_without_name' => false,
        'phpdoc_no_package'       => false,
        'concat_space'            => [
            'spacing' => 'one',
        ],
        'binary_operator_spaces' => [
            'align_double_arrow' => true,
            'align_equals'       => true,
        ],
    ))
    ->setFinder($finder)
;
```

Input:
```php
$templating='a';
$subject   = 'b';
$subject  ='c';
```

Output:
```php
$templating='a';
$subject   = 'b';
$subject   ='c';
```

Expected output (missing whitespace around equal sign):
```php
$templating = 'a';
$subject    = 'b';
$subject    = 'c';
```

Possible reason:
The logic for adding whitespace around the operater in BinaryOperatorSpacesFixer.php is at the end of `fixWhiteSpaceAroundOperator`, but when `align_equals` is enabled, the function returns in line 136 and forwards to `AlignEqualsFixerHelper`. This fixer won't inject additional whitespace around the equal sign.

**Comment by NewFuture** [comment]
meet the same bug

**Comment by kaiyulee** [comment]
also with `align_double_arrow`

**Comment by keradus** [comment]
Closes due to merge of #2573
